ui/src/app/utils/access-utils.spec.tsx

```diff
@@ -928,3 +929,54 @@ describe(isCompleted.name, () => {
     expect(isCompleted(status, duccSignedVersion)).toBeFalsy();
   });
 });
+
+describe(syncModulesExternal.name, () => {
+  beforeEach(() => {
+    registerApiClient(ProfileApi, new ProfileApiStub());
+    serverConfigStore.set({
+      config: {
+        ...defaultServerConfig,
+      },
+    });
+  });
+
+  it('should return a successful promise when no modules are passed', () => {
+    expect(syncModulesExternal([])).toBeTruthy();
```

Comment:
I think these are only checking that a promise exists, and we actually want `.resolves.toBeTruthy()`

Reply:
Good catch! I updated it.